skelebot/components/repository/s3Repo.py

```diff
@@ -94,6 +94,7 @@ def pull(self, artifact, version, currentVersion=None, override=False, user=None
             obj = client.get_object(Bucket=bucket, Key=path)
             obj = obj['Body'].read()
         else:
+            print(f"{bucket} {path} {dest}")
```

Comment:
debug message?
Otherwise we can use the `INFO` prefix from `common.py` to make the message a little nicer to the user.

Reply:
Nah, debug, I'll delete it